pkg/gitops/tagging.go

```diff
@@ -24,25 +24,25 @@ import (
 	"k8s.io/klog/v2"
 )
 
-func (cr *CustomRepo) TagCommit(commit_sha string, tag string, tagger *object.Signature) error {
```

Comment:
we use camel case variables by convention in go. s/commit_sha/commitSha

I didn't check if a similar change was needed for other variable names